internal/byok/command_create.go

```diff
@@ -184,12 +249,24 @@ func renderAzureEncryptionPolicy(key byokv1.ByokV1Key) (string, error) {
 	return strings.Join(az, "\n"), nil
 }
 
+func renderGCPEncryptionPolicy(key byokv1.ByokV1Key) string {
+	// No need to do a sanity check for this key as it is handled by the BYOK API validation
+	// assumed to be a valid key ID
+	splitKeyId := strings.Split(key.Key.ByokV1GcpKey.KeyId, "/")
+	project, location, keyRing, keyName, group := splitKeyId[1], splitKeyId[3], splitKeyId[5], splitKeyId[7], key.Key.ByokV1GcpKey.GetSecurityGroup()
+
+	encryptionPolicyMetadata := newGcpPolicyMetadata(project, location, keyRing, keyName, group)
+	return encryptionPolicyMetadata.renderPolicy()
+}
+
 func getPostCreateStepInstruction(key byokv1.ByokV1Key) string {
 	switch {
 	case key.Key.ByokV1AwsKey != nil:
 		return `Copy and append these permissions into the key policy "Statements" field of the ARN in your AWS key management system to authorize access for your Confluent Cloud cluster.`
 	case key.Key.ByokV1AzureKey != nil:
 		return "To ensure the key vault has the correct role assignments, please run the following azure-cli command (certified for azure-cli v2.45):"
+	case key.Key.ByokV1GcpKey != nil:
+		return "To ensure the key has the correct role assignments, please run the following gcloud-cli command:"
```

Comment:
```suggestion
		return "To ensure the key has the correct role assignments, please run the following `gcloud` CLI command:"
```

Reply:
I'd like to keep it somewhat consistent with the above instruction for Azure.
What do you think about just using `Azure CLI` and `Google Cloud CLI`?

Reply:
> What do you think about just using Azure CLI and Google Cloud CLI?

Sure, that works for me. I just want to avoid `gcloud-cli` since that's not a real command.

Reply:
Regarding consistency between the Azure and the GCP case: Currently, we advise that the Azure command is 
> (certified for azure-cli v2.45)
I'm in favor of getting rid of this statement. If we want to keep some information on what version we ran this statement against, could we perhaps move the information to the BYOK documentation? What do you think?

Reply:
In retrospect, we could probably change this to say "latest version of v2", but I'm hesitant to remove this entirely, since Microsoft could choose to remove or rename this command in v3, and I don't want to have to wait for a customer to tell us that it's broken to fix it.